Source/Charts/Charts/PieChartView.swift

```diff
@@ -659,4 +662,28 @@ open class PieChartView: PieRadarChartViewBase
             }
         }
     }
+    
+    /// The offset value that is used for shifting the data label outside of the center of chart.
+    /// **default**: 0.0
+    open var dataLabelOffset: CGFloat
+    {
+        get
+        {
+            return _dataLabelOffset
+        }
+        set
+        {
+            _dataLabelOffset = newValue
+            
+            if _dataLabelOffset > 1.0
```

Comment:
any reason to lock at 1.0? how about 2.0?

Reply:
Or 1000?

Reply:
it's a multiplier, I guess we need to let the consumer knows this so they won't write 1000x..

Reply:
I don't think it should be though. It seems very unintuitive.

Reply:
1.0 is max.
like 0 to 100 i.e 0.0 to 1.0